app/partials/actions/add.html

```diff
@@ -1,10 +1,10 @@
 <div class="container">
 	<h3>
-		Add {{rdesc.name.singular}}
+		Add {{rdesc.attributes.name.singular}}
 	</h3>
-	<h5>{{rdesc.description}}</h5>
+	<h5>{{rdesc.attributes.description}}</h5>
 
-	<div class="form-group" ng-repeat="field in rdesc.fields" ng-include="'/partials/fields/default-field.html'">
+	<div class="form-group" ng-repeat="field in rdesc.attributes.fields" ng-include="'/partials/fields/default-field.html'">
```

Comment:
@maxdumas Ok, I see how `independent` doesn't work. My trouble with `automatic` is that, while it makes sense from the client's perspective, it doesn't make sense from the server's perspective (since the server may need to write logic for, e.g. generating ids or updating modified dates). 

Now, you could argue that the client's perspective is much more important for names in the documentation, since the docs are client facing, and I'd totally agree. The problem is that something has to be set on the server to generate that documentation output, and it's weird if that thing in the server-side code is named from the client's perspective (and even more confusing/way overly complicated if there's a mapping from the server setting's name to the name in the docs).

So, in an ideal world, I'd want something "perspectiveless", like "required" and "friendlyName" already are. Along those lines, what do you think of `"server-managed"`? It's longer, but has the benefit of being utterly unambiguous.